src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -123,6 +123,7 @@ public static class Features
             public const string CodeModelMethodXml = "CodeModel.MethodXml";
             public const string CommentSelection = nameof(CommentSelection);
             public const string Completion = nameof(Completion);
+            public const string ConvertAutoPropertyToFullProperty = nameof(ConvertAutoPropertyToFullProperty);
```

Comment:
This is a code action right?  Should probably go up in the code action section.  (but i don't care that much :))